src/api/controllers/TeamController.js

```diff
@@ -120,6 +121,78 @@ module.exports = {
       });
   },
 
+  getAll: (_, res) => {
+    var response = [];
```

Comment:
Similarly to my previous comment, this can be changed to a **const**.
```suggestion
    const response = [];
```